app/router.js

```diff
@@ -42,6 +42,7 @@ Router.map(function() {
   this.route('public', { path: '/e/:event_id' }, function() {
     this.route('sessions', function() {
       this.route('list', { path: '/s/:session_status' });
```

Comment:
/session/, not /s/

Reply:
Should I remove `/s`? Because after that the route will become `sessions/:session_status`

Reply:
Yes

Reply:
Done. Please review